python/pyspark/mllib/linalg/distributed.py

```diff
@@ -500,6 +661,25 @@ def numCols(self):
         """
         return self._java_matrix_wrapper.call("numCols")
 
+    def transpose(self):
+        """
+        Transpose this CoordinateMatrix.
+
+        >>> entries = sc.parallelize([MatrixEntry(0, 0, 1.2),
+        ...                           MatrixEntry(1, 0, 2),
+        ...                           MatrixEntry(2, 1, 3.7)])
+        >>> mat = CoordinateMatrix(entries)
+        >>> mat_transposed = mat.transpose()
+
```

Comment:
Yeah, I like the visual clarity when viewing these tests on the Python docs, as it helps indicate that the following two tests rely on the data structures formed above.  This is generally the pattern I've followed with these classes for cases with >1 test.